Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -118,9 +121,35 @@ public struct Aside {
                 todo,
                 version,
                 `throws`,
+                seeAlso,
             ]
         }
         
+        /// The heading text to use when rendering this kind of aside.
+        ///
+        /// For multi-word asides this value may differ from the aside's ``rawValue``.
+        /// For example, the ``seeAlso`` aside's `rawValue` is `"SeeAlso"` but its
+        /// `displayName` is `"See Also"`.
+        /// Likewise, ``nonMutatingVariant``'s `rawValue` is
+        /// `"NonMutatingVariant"` and its `displayName` is `"Non-Mutating Variant"`.
+        ///
+        /// For simpler, single-word asides like ``bug``, the `displayName` and `rawValue` will
+        /// be the same.
+        public var displayName: String {
```

Comment:
Swift-DocC currently implements this here: https://github.com/apple/swift-docc/blob/9c28bc4391e04bc6eaf4edf6226e1b94ee93805e/Sources/SwiftDocC/Model/Rendering/Content/RenderBlockContent.swift#L298.

I'm not sure it makes sense for Swift-Markdown to own the asides Swift-DocC knows how to parse but since that's the current state I think it makes sense for it to _also_ own the custom `displayName` values.

Reply:
All absolutely fair statements. This was a difficult choice in the beginning but it just seemed too useful to other clients.

Reply:
> This was a difficult choice in the beginning but it just seemed too useful to other clients.

That's a good point. Definitely see the value in Swift-Markdown being able to do this as well. It seems like we may have gone back and forth on it to some extent – I noticed there's also this simple list item extractor in Swift-DocC that's currently unused: https://github.com/apple/swift-docc/blob/9c28bc4391e04bc6eaf4edf6226e1b94ee93805e/Sources/SwiftDocC/Utility/MarkupExtensions/ListItemExtractor.swift#L18

Reply:
It does seem odd to me that there's a split concern between Swift-Markdown and Swift-DocC, but it also seems like the list-item extractor in docc is mainly for extracting "Parameters" and "Returns" clauses, with it mostly deferring to Swift-Markdown for generic asides. (Though it does scan that list you linked, it seems like the only thing that happens is that list-items with those tags are dropped from the output entirely.)